tcms/core/utils/__init__.py

```diff
@@ -60,7 +60,7 @@ def clean_request(request, keys=None):
     """
```

Comment:
``clean_request`` could be moved to ``testcases``, and also you are right, many URLs are not named. These would be fixed in other PRs.

> From what I can tell this entire function can be removed. Also the section in open_run_get_case_runs() where we sort the existing tcrs can be removed. Objections ?

I guess you mean to remove sorting by sortkey, right? If yes, my answer is no. Sorting by sortkey is a required feature for Nitrate. It cannot be removed from Nitrate now.